README.md

```diff
@@ -113,39 +115,40 @@ module.exports = [
 			"json/no-duplicate-keys": "error",
 		},
 	},
-];
+]);
 ```
 
 ## Recommended Configuration
 
 To use the recommended configuration for this plugin, specify your matching `files` and then use the `json.configs.recommended` object, like this:
 
 ```js
+import { defineConfig } from "eslint/config";
 import json from "@eslint/json";
 
-export default [
+export default defineConfig([
 	// lint JSON files
 	{
 		files: ["**/*.json"],
 		ignores: ["package-lock.json"],
 		language: "json/json",
-		...json.configs.recommended,
+		extends: ["json/recommended"],
 	},
 
 	// lint JSONC files
 	{
 		files: ["**/*.jsonc"],
 		language: "json/jsonc",
-		...json.configs.recommended,
+		extends: ["json/recommended"],
 	},
 
 	// lint JSON5 files
 	{
 		files: ["**/*.json5"],
 		language: "json/json5",
-		...json.configs.recommended,
+		extends: ["json/recommended"],
 	},
-];
+]);
```

Comment:
This results in `TypeError: Plugin "json" not found.`.

For `extends` from a plugin config to work, we'd need to specify the plugin in the same config object where `extends` is.